### PR TITLE
Upstream merge 8/20

### PR DESCRIPTION
**Pull request by willrlin**
## Issue

- Addresses #293 
- [x] This change is a dependency for another issue

## Description

Upstream merge to address the backend/core translation fix.

### CODE REVIEW

File: backend/core/package.json [hunk 1]
```diff
@@ -130,5 +130,6 @@
   "engines": {
     "node": "12",
     "yarn": "^1.22"
-  }
+  },
+  "gitHead": "5f7694e3f2baff2a1a7a22e0676e487dda781a3e"
```
Comment (abbiefarr) [inline backend/core/package.json:134]: I'm not sure this was intentionally added upstream. I found [this commit](https://github.com/bloom-housing/bloom/commit/0cbc5401ea9fd1d89623c499b3ed1f27bee8f98f) via blame, but there wasn't a PR linked to it for some reason.

I couldn't find much documentation about this issue, but I did find [some people complaining](https://github.com/lerna/lerna/issues/1880) about this happening accidentally with lerna, which Bloom uses.

File: ui-components/package.json [hunk 2]
```diff
@@ -94,5 +94,6 @@
     "react-transition-group": "^4.4.1",
     "tailwindcss": "^1.8.12",
     "typesafe-actions": "^5.1.0"
-  }
+  },
+  "gitHead": "5f7694e3f2baff2a1a7a22e0676e487dda781a3e"
```
Comment (abbiefarr) [inline ui-components/package.json:98]: Ditto here

**Review by abbiefarr — APPROVED** [review]

File: backend/core/package.json (hunk 1 above)
Comment (willrlin) [reply backend/core/package.json]: Done!

File: ui-components/package.json (hunk 2 above)
Comment (willrlin) [reply ui-components/package.json]: Done!

**Pull request merged**